### PR TITLE
Add info about the props limit

### PR DESCRIPTION
**Pull request by lemieux**
This PR adds info about the new custom props size limitation.

I also included a small change to the conversation extensions section to reflect the last fix, and I fixed the url on the logo in the REST docs as `https://www.smooch.io` doesn't actually work for a redirect (since it's S3 handling it, it only does in on http).

### CODE REVIEW

File: pages/guide/conversation-extensions.md [hunk 1]
```diff
@@ -162,16 +162,19 @@ To use the Webview SDK, add the following script tag to your html page:
         js.id = id;
         js.src = 'https://cdn.smooch.io/webview-sdk.min.js';
         fjs.parentNode.insertBefore(js, fjs);
-    }(document, 'script', 'WebviewSdk'));
+    }(document, 'script', 'WebviewSdkScript'));
```
Comment (jpjoyal) [inline pages/guide/conversation-extensions.md]: Why is this changing ? Do people need to change their snippet ?

Comment (lemieux) [reply pages/guide/conversation-extensions.md]: We got a bug report about it, I don't think people need to make that change, but it's just to be safe. https://github.com/smooch/webview-date-picker/issues/2

Comment (lemieux) [reply pages/guide/conversation-extensions.md]: The lib should be taking over that variable at the end, this is just to make it less confusing.

File: pages/guide/managing-user-information.md [hunk 2]
```diff
@@ -167,6 +167,10 @@ Smooch.updateUser({
 
 You can also specify any other kind of profile information that will be sent along when users contact you.
 
+<aside class="notice">
+Profile properties are limited to a global size of 4KB. An error will be returned if an app user's profile properties are going over the size limit.
```
Comment (jpjoyal) [inline pages/guide/managing-user-information.md]: Didn't we have this disclaimer in the REST doc too ? I can't find it anymore. I believe it was explaining what you mean by "global" here. I would rephrase to "Profile properties are limited to 4KB per users. Both key and values are accounted for in this size limitation"

Comment (lemieux) [reply pages/guide/managing-user-information.md]: https://github.com/smooch/smooch-docs/pull/407/files#diff-97ee64acb842e1d89e2e78afdc01ae17R16

Comment (lemieux) [reply pages/guide/managing-user-information.md]: I'll update both disclaimers with what you suggested.
